### PR TITLE
Fe view project

### PR DESCRIPTION
**Pull request by Mohammadhasham1999**
Fixed review comments for view project

### CODE REVIEW

File: src/ProjectIT/Client/Components/ProjectDetailsView/ProjectDetailsView.razor.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+using Microsoft.AspNetCore.Components;
+
+
```
Comment (joac1144) [inline src/ProjectIT/Client/Components/ProjectDetailsView/ProjectDetailsView.razor.cs:3]: Only 1 free line between usings and namespace

File: src/ProjectIT/Client/Components/ProjectDetailsView/ProjectDetailsView.razor.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+namespace ProjectIT.Client.Components.ProjectDetailsView;
+
+
```
Comment (joac1144) [inline src/ProjectIT/Client/Components/ProjectDetailsView/ProjectDetailsView.razor.cs:6]: Only 1 free line between namespace and class

File: src/ProjectIT/Client/Components/ProjectDetailsView/ProjectDetailsView.razor.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+public partial class ProjectDetailsView
+{
+    [Parameter]
+    public string? Title { get; set;}
+
+    [Parameter]
+    public string? Description { get; set;}
+
```
Comment (joac1144) [inline src/ProjectIT/Client/Components/ProjectDetailsView/ProjectDetailsView.razor.cs:14]: Fix whitespace here and all other places

File: src/ProjectIT/Client/Pages/ProjectDetails.razor [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+@if(Project == null) 
+{
+    <div class="spinner-border spinner" role="status">
+        <span class="visually-hidden">Loading...</span>
+    </div>
+} 
+else 
+{
+
```
Comment (joac1144) [inline src/ProjectIT/Client/Pages/ProjectDetails.razor:26]: whitespace

File: src/ProjectIT/Client/Pages/ProjectDetails.razor [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+            <div>
+                <Panel Title="Project Info">
+                    <div class="d-flex flex-column" style="">
+                        Programme: <br>
+                        <div class="d-flex programmes">
+                            @foreach (Programme programme in Project.Programmes)
+                            {
+                                <p1 class="fw-normal"> @programme </p1>
+                                <span class="border-start border-2 border-dark border-opacity-10 mx-2" />
+
```
Comment (joac1144) [inline src/ProjectIT/Client/Pages/ProjectDetails.razor:38]: whitespace

File: src/ProjectIT/Client/Pages/ProjectDetails.razor [hunk 6]
```diff
@@ -0,0 +1,88 @@
+@page "/projectDetails" /* Fix this to be {Id:int} later */
```
Comment (joac1144) [inline src/ProjectIT/Client/Pages/ProjectDetails.razor:1]: Why later?

File: src/ProjectIT/Client/Pages/ProjectDetails.razor.cs [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+namespace ProjectIT.Client.Pages;
+
+public partial class ProjectDetails
+{
+    //TODO: for later use, don't delete
+    [Parameter]
+    public int Id { get; set; }
+
+    private ProjectDetailsDto? Project;
```
Comment (joac1144) [inline src/ProjectIT/Client/Pages/ProjectDetails.razor.cs:16]:
```suggestion
    private ProjectDetailsDto? project;
```

File: src/ProjectIT/Client/Pages/ProjectDetails.razor.cs [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+    public IList<FilterTag> Tags { get; set; } = new List<FilterTag>();
+
+    private void ApplyProject(NavigationManager navigationManager)
+    {
+        navigationManager.NavigateTo("/");
+    }
+
+    protected override async Task OnInitializedAsync()
+    {
+        Project = await HttpClient.GetFromJsonAsync<ProjectDetailsDto>("https://localhost:7094/projects/1");
```
Comment (joac1144) [inline src/ProjectIT/Client/Pages/ProjectDetails.razor.cs:29]:
```suggestion
        Project = await HttpClient.GetFromJsonAsync<ProjectDetailsDto>("projects/{id}");
```
should work

File: src/ProjectIT/Client/Pages/ProjectDetails.razor.scss [hunk 9]
```diff
@@ -0,0 +1,15 @@
+.projectDetailsView {
```
Comment (joac1144) [inline src/ProjectIT/Client/Pages/ProjectDetails.razor.scss:1]: Change name to follow CSS class name standards (snake case but dashes, e.g. project-details-view).

File: src/ProjectIT/Client/Pages/ProjectDetails.razor.scss [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+    gap: 50px;
+}
+
+.leftMostPanels {
```
Comment (joac1144) [inline src/ProjectIT/Client/Pages/ProjectDetails.razor.scss:5]: Why have both a leftMostPanels and a rightMostPanels if they have the same styling anyway?
Also change name to something more saying, e.g. `panel-gap`

File: src/ProjectIT/Client/Pages/TestPage.razor [hunk 11]
```diff
@@ -3,15 +3,18 @@
...
 
-<PageTitle>A test page</PageTitle>
-
-this a test page
-
-Here are some components:
 
+<PageTitle>A test page</PageTitle>
 <SearchField PlaceholderText="Some placeholder text..." />
+<div>
```
Comment (joac1144) [inline src/ProjectIT/Client/Pages/TestPage.razor:16]: Github Actions says "Unclosed tag 'div' with no matching end tag."

File: src/ProjectIT/Client/Pages/TestPage.razor [hunk 12]
```diff
@@ -3,15 +3,18 @@
...
+    <div>
```
Comment (joac1144) [inline src/ProjectIT/Client/Pages/TestPage.razor:17]: Github Actions says "Unclosed tag 'div' with no matching end tag."

File: src/ProjectIT/Client/Pages/TestPage.razor [hunk 13]
```diff
@@ -36,4 +38,5 @@ Here are some components:
 @if (GptResult != null)
 {
     @GptResult
-}
\ No newline at end of file
+}
+
```
Comment (joac1144) [inline src/ProjectIT/Client/Pages/TestPage.razor:42]: Delete whitespace

**Review by joac1144 — CHANGES_REQUESTED** [review]

File: src/ProjectIT/Client/Pages/ProjectDetails.razor (hunk 6 above)
Comment (Mohammadhasham1999) [reply src/ProjectIT/Client/Pages/ProjectDetails.razor:1]: Fordi det giver mening. Lidt svært at forklare.

File: src/ProjectIT/Client/Pages/ProjectDetails.razor.cs (hunk 8 above)
Comment (Mohammadhasham1999) [reply src/ProjectIT/Client/Pages/ProjectDetails.razor.cs:29]: Doesn't. Eventually one can put the Uri (i.e. https://localhost:7094) in appsettings.json?

**Pull request closed without merging**